ddl/index.go

```diff
@@ -1072,6 +1072,18 @@ func (w *addIndexWorker) batchCheckUniqueKey(txn kv.Transaction, idxRecords []*i
 	return nil
 }
 
+func encodeHandleInUniqueIndexValueForBatchCheck(h kv.Handle) []byte {
```

Comment:
We can modify `tables.EncodeHandleInUniqueIndexValue` to support common handle and use it there https://github.com/pingcap/tidb/blob/master/table/tables/index.go#L373